src/main/java/graphql/GraphQL.java

```diff
@@ -29,16 +29,69 @@
 
     private static final Logger log = LoggerFactory.getLogger(GraphQL.class);
 
```

Comment:
I think this should take into account the mutation strategy selection, which is already released https://github.com/graphql-java/graphql-java/pull/240/files